### PR TITLE
js_task_generate_table_DOM

### PR DESCRIPTION
**Pull request by AndriiYelieva**
done, but tests not work
    - [DEMO LINK](https://AndriiYelieva.github.io/js_task_generate_table_DOM/)
https://AndriiYelieva.github.io/js_task_generate_table_DOM/

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -5,4 +5,30 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const current = document.querySelector('tr');
+
+people.forEach((person) => {
```
Comment (VitaliyBondarenko1982) [inline src/scripts/main.js:10]: use destructuring
```suggestion
people.forEach(({name: personName, died, born, sex }) => {
```

File: src/scripts/main.js [hunk 2]
```diff
@@ -5,4 +5,30 @@ const people = require('./lib/people');
...
+  person.age = person.died - person.born;
+  person.century = Math.ceil(person.died / 100);
+
+  if (person.sex === 'f') {
+    person.sex = 'female';
+  } else {
```
Comment (VitaliyBondarenko1982) [inline src/scripts/main.js:16]: redundant to use if else. better to use ternary operator
Female and Male should be with Capital first letter

File: src/scripts/main.js [hunk 3]
```diff
@@ -5,4 +5,30 @@ const people = require('./lib/people');
...
+
+  current.parentElement.append(table);
+
+  table.insertAdjacentHTML(
+    'beforeend', `
+    <th>${person.name}</th>
+    <th>${person.sex}</th>
+    <th>${person.born}</th>
+    <th>${person.died}</th>
+    <th>${person.age}</th>
```
Comment (VitaliyBondarenko1982) [inline src/scripts/main.js:30]:
```suggestion
    <th>${died - born}</th>
```

File: src/scripts/main.js [hunk 4]
```diff
@@ -5,4 +5,30 @@ const people = require('./lib/people');
...
+  }
+
+  const table = document.createElement('tr');
+
+  current.parentElement.append(table);
+
+  table.insertAdjacentHTML(
+    'beforeend', `
+    <th>${person.name}</th>
+    <th>${person.sex}</th>
```
Comment (VitaliyBondarenko1982) [inline src/scripts/main.js:27]:
```suggestion
    <th>${sex === 'f' ? 'Female' : 'Male'}</th>
```

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]

File: src/scripts/main.js [hunk 5]
```diff
@@ -5,4 +5,23 @@ const people = require('./lib/people');
...
+const current = document.querySelector('tr');
+
+people.forEach(({ name: fullName, died, born, sex }) => {
+  const century = Math.ceil(died / 100);
+
+  const table = document.createElement('tr');
+
+  current.parentElement.append(table);
+
+  table.insertAdjacentHTML(
```
Comment (etojeDenys) [inline src/scripts/main.js:17]: Use 'td' elements for table data instead of 'th'. 'th' elements are for table headers.

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**